packages/trader/src/Modules/Trading/Components/Form/form-layout.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Loadable from 'react-loadable';
+import { isMobile } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+import { useTraderStore } from 'Stores/useTraderStores.js';
+
+type TFormLayout = {
+    is_market_closed: ReturnType<typeof useTraderStore>['is_market_closed'];
+    is_trade_enabled: boolean;
```

Comment:
`is_trade_enabled` also comes from `useTraderStore()`, maybe it's possible to use  `ReturnType<typeof useTraderStore>['...'] `here? Just asking, boolean is also ok